index.html

```diff
@@ -174,7 +174,57 @@ <h4 class="about">About me</h4>
         </ul>
     </div>
         </section>
+
+        <section class="form">
+            <h2 class="prg">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+          
+                <form  action="https://formspree.io/f/xpzbqzab" method="post">
+                    <input
+                    class="input"
+                    type="text"
+                    name="name"
+                    id="name"
+                    placeholder="Full name"
+                    maxlength="30"
+                    required/>
+                
+                <hr>
+                
+                  <input
+                    class="input"
+                    type="email"
+                    name="email"
+                    id="email"
+                    placeholder="Email address"
+                    required
+                  />
+                
+                <hr>
+                
+                  <textarea class="msg"
+                    name="message"
+                    id="message"
+                    cols="30"
+                    rows="10"
+                    placeholder="Enter text here"
+                    maxlength="500"  
+                    required   
+                  ></textarea>
+                </form>
+                
+                <div class="tonic">
+                    <button class="btn-tonic" type="submit">Get In Touch</button>
```

Comment:
- [ ] In relation to what the previous reviewer requested. I think it will be good to follow the button interaction from the template. When someone hovers the button the following should interaction should take effect:
     -  The background colour should change
     - The button should have a border radius. Please take note that the border-radius colour and text colour also

See Design Guide    |
:--------------------------:|
![](https://user-images.githubusercontent.com/85113567/149190313-c107573b-bfb5-41e9-b9cc-a6486ff84550.jpg)